Specs/DateTime/German/DatePeriodExtractor.json

```diff
@@ -1,12 +1,12 @@
 [
   {
-    "Input": "Ich werde diesen Januar nicht hier sein.",
+    "Input": "Diesen Januar werde nicht hier sein.",
     "NotSupported": "javascript, python",
     "Results": [
       {
-        "Text": "diesen Januar",
+        "Text": "Diesen Januar",
```

Comment:
All expected output should be in lowercase as sentence case is not preserved internally in the recognizers.

Reply:
In this case it was a mistake as even the input sentence hat the lover case word. But I just checked the English specs and in general the expected output text matches the case of the input here.

  {
    "Input": "I'll be out month of May",
    "Results": [
      {
        "Text": "month of May",
        "Type": "daterange",
        "Start": 12,
        "Length": 12
      }
    ]
  },